test/batcher_test.go

```diff
@@ -0,0 +1,366 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
```

Comment:
please make this look like in the rest of the files
with an empty line in between